src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -322,17 +321,19 @@ private constructor(
           // in the plugin directory.
           Files.deleteIfExists(binaryTarget)
         }
-        logger.info("extracting Node binary to " + binaryTarget.toAbsolutePath())
+        logger.info("Extracting Node binary to " + binaryTarget.toAbsolutePath())
         Files.copy(binarySource, binaryTarget, StandardCopyOption.REPLACE_EXISTING)
         val binary = binaryTarget.toFile()
         if (binary.setExecutable(true)) {
           binary
         } else {
-          throw CodyAgentException("failed to make executable " + binary.absolutePath)
```

Comment:
That was failing on CI, where apparently we cannot make processes executable.
That said I think fallback to the base node process is also sensible default for end users.